ts/components/ContextualHelp/CheckboxFormEntry.tsx

```diff
@@ -0,0 +1,91 @@
+import { View } from "native-base";
+import React from "react";
+import { StyleSheet } from "react-native";
+
+import I18n from "../../i18n";
+
+import { IOStyles } from "../core/variables/IOStyles";
+import { RawCheckBox } from "../core/selection/RawCheckBox";
+import { Label } from "../core/typography/Label";
+import Accordion from "../ui/Accordion";
+
+/**
+ * Checkboxes we need as of now
+ */
+export enum CheckboxIDs {
+  sendScreenshot = "sendScreenshot",
+  sendPersonalInfo = "sendPersonalInfo"
+}
+
+type CheckboxLabelMap = {
+  [key in CheckboxIDs]: {
+    cta: string;
+    accordion?: {
+      title: string;
+      content: string;
+    };
+  };
+};
+
+const checkboxStyle = StyleSheet.create({
+  container: {
+    flexDirection: "row",
+    alignItems: "flex-start",
+    justifyContent: "space-evenly"
+  }
+});
+
+type Props = {
+  target: CheckboxIDs;
+  isChecked?: boolean;
+  onToggle: () => void;
+};
+
+const checkboxLabelMapFactory = (): CheckboxLabelMap => ({
+  sendScreenshot: {
+    cta: I18n.t("contextualHelp.sendScreenshot.cta")
+  },
+  sendPersonalInfo: {
+    cta: I18n.t("contextualHelp.sendPersonalInfo.cta"),
+    accordion: {
+      title: I18n.t("contextualHelp.sendPersonalInfo.informativeTitle"),
+      content: I18n.t("contextualHelp.sendPersonalInfo.informativeDescription")
+    }
+  }
+});
+
+/**
+ * Checkbox+Label+Optional Accordion component
+ */
+const CheckboxFormEntry: React.FC<Props> = ({
+  target,
+  isChecked,
+  onToggle
+}) => {
+  const checkboxLabelKeys = checkboxLabelMapFactory();
+
+  return (
+    <View style={checkboxStyle.container}>
+      <RawCheckBox checked={isChecked} onPress={onToggle} />
+      <View hspacer />
+      <View style={IOStyles.flex}>
+        <Label
+          testID="ContextualHelpCheckboxFormEntryLabel"
+          color={"bluegrey"}
+          weight={"Regular"}
+          onPress={onToggle}
+        >
+          {checkboxLabelKeys[target].cta}
+        </Label>
+        {"accordion" in checkboxLabelKeys[target] && (
+          <Accordion
+            title={checkboxLabelKeys[target].accordion?.title || ""}
+            content={checkboxLabelKeys[target].accordion?.content || ""}
+          />
+        )}
```

Comment:
```suggestion
        const accordion = checkboxLabelKeys[target].accordion;
        [...]
        {accordion && (
          <Accordion title={accordion.title} content={accordion.content} />
        )}
```
Could it be an improvement?
In this way we avoid 2 optional chaining `?.` and 2 logical OR `||`
IMHO using several times the optional chaining could decrease code performance
![Schermata 2021-08-04 alle 11 33 16](https://user-images.githubusercontent.com/822471/128158415-7118fd81-83b5-427b-a897-88a642c6a991.jpg)

Reply:
Very good point! I was copy-pasting blindly and missed this low-hanging fruit.
Fixed [as suggested](https://github.com/pagopa/io-app/pull/3266/commits/e4e9abf83b7eb1efa3b1a75a66ebf4e12c6fe423#diff-a3d54657eb55645a2e5cebccb5308f137cfaa6d26502e788ba303aac28c260daR82)